CONTRIBUTION.md

```diff
@@ -27,6 +27,16 @@ We are very happy that you consider implementing algorithms and data structure f
 - If you have modified/added code work, make sure the code compiles before submitting.
 - **Be consistent in use of there guidelines when submitting**
 
+#### New File Name guidelines
+- Use lowercase words with ``"_"`` as separator
+- For instance 
+```
+MyNewCppClass.cpp       is incorrect
```

Comment:
Let's make this __MyNewCppClass.CPP__ because we also care about the file extension (.cpp).

Reply:
Yes, agreed